extensions/VisualEditor/wikia/modules/ve/init/ve.init.wikia.Optimizely.js

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Utility functions for running Optimizely A/B tests with VisualEditor
+ */
+
+( function () {
+	/**
+	 * @returns {number} The A/B test variant number for the Toolbar A/B test
+	 */
+	ve.init.wikia.getToolbarABTestVariantNumber = function () {
```

Comment:
It might be the closure around the contents of the file which is confusing. I was just emulating code style from elsewhere, and it isn't required here since ve.init.wikia is set in ve.init.wikia.js which is loaded before anyway. I'll remove it.